src/applications/vaos/components/Breadcrumbs.jsx

```diff
@@ -19,9 +19,9 @@ export default function VAOSBreadcrumbs({ children }) {
 
   useEffect(
     () => {
-      setBreadcrumb(newLabel);
+      setBreadcrumb(urlLabel || newLabel);
```

Comment:
Instead of this change line 18 to `const newLabel = labelOverride || label || covidLabel;`